src/m365/todo/commands/task/task-add.ts

```diff
@@ -102,6 +125,18 @@ class TodoTaskAddCommand extends GraphCommand {
           return `'${args.options.reminderDateTime}' is not a valid ISO date string`;
         }
 
+        if (args.options.completedDateTime && !validation.isValidISODateTime(args.options.completedDateTime)) {
+          return `'${args.options.completedDateTime}' is not a valid datetime.`;
+        }
+
+        if (args.options.startDateTime && !validation.isValidISODateTime(args.options.startDateTime)) {
+          return `'${args.options.startDateTime}' is not a valid datetime.`;
+        }
+
+        if (args.options.status && TodoTaskAddCommand.allowedStatuses.indexOf(args.options.status) < 0) {
+          return `${args.options.status} is not a valid value for status. Valid values are ${TodoTaskAddCommand.allowedStatuses.join(', ')}`;
+        }
```

Comment:
Can we make this not case-sensitive? When they pass e.g. `Completed`, it gets validated and passed to the API as `completed`